packages/eslint-plugin-ckeditor5-rules/lib/rules/no-relative-imports.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+	meta: {
+		type: 'problem',
+
+		docs: {
+			description: 'Disallow relative imports from CKEditor5 packages',
+			category: 'CKEditor5'
+		},
+		fixable: 'code',
+		schema: [] // no options
+	},
+	create( context ) {
+		return {
+			ImportDeclaration: node => {
+				const importPath = node.source.value;
+
+				if ( importPath.startsWith( '../../ckeditor5-' ) ) {
```

Comment:
What if an import will start with `../../../*`? There should be a smarter solution to check it.

Reply:
☝️ This will probably happen in the CF repository where a path looks like `packages/@ckeditor/ckeditor5-engine/...`.

Reply:
The second case – this replace will fail in a package that uses relative imports for importing other files from the same package (but they are nested):

```
ckeditor5-bar/src/
  |-- foo/
  |-- |-- ...
  |-- bar/
  |-- |-- foo/
  |-- |-- |-- nested/
  |-- |-- |-- |-- ...
```

Imports from `foo/*` inside `bar/foo/nested/` will require `../../foo/*` which trigger the plugin. So we need to ensure that relative imports ([even `../../../` and `../../../../`](https://github.com/ckeditor/ckeditor5-ui/blob/9aaad899d81828de6d44282f81ad2e2b4f307282/tests/manual/panel/sticky/stickypanelview.js#L6-L7)) inside single package won't be replaced.

Reply:
They won't since: `/ckeditort-` is checked also not a relative path per se.